tailcfg/tailcfg.go

```diff
@@ -819,6 +820,7 @@ type Hostinfo struct {
 	Userspace       opt.Bool       `json:",omitempty"` // if the client is running in userspace (netstack) mode
 	UserspaceRouter opt.Bool       `json:",omitempty"` // if the client's subnet router is running in userspace (netstack) mode
 	AppConnector    opt.Bool       `json:",omitempty"` // if the client is running the app-connector service
+	ServicesHash    string         `json:",omitempty"` // SHA256 hash of the most recent list of tailnet services, change in hash indicates config should be fetched via c2n
```

Comment:
I thought we were just going to say this is an ETag-like thing and not specify the hash?

Reply:
Yep, this is a leftover, my bad